airbyte-integrations/connectors/source-outreach/source_outreach/manifest.yaml

```diff
@@ -180,7 +180,7 @@ definitions:
 
   requester:
     type: HttpRequester
-    url_base: https://api.outreach.io/api/v2
+    url_base: https://api.outreach.io/api/v2/
```

Comment:
Slightly weird — were you able to test this, @btkcodedev? I assumed that the CDK would attempt and add / in automaticallly. But let's see!

Reply:
Yes @natikgadzhi, I've tested it with both CLI and builder, it was the reason. 🎊